packages/cli/test/integration.test.ts

```diff
@@ -138,7 +137,7 @@ async function vcLink(projectPath) {
     }
   );
 
-  expect(exitCode).toBe(0);
+  expect(exitCode, formatOutput({ stdout, stderr })).toBe(0);
 }
 
 // AVA's `t.context` can only be set before the tests,
```

Comment:
Mention of AVA in the comment caught my eye.

Reply:
Thanks! https://github.com/vercel/vercel/pull/9671